fdbserver/DataDistribution.actor.cpp

```diff
@@ -2766,6 +2770,30 @@ ACTOR Future<Void> serverTeamRemover(DDTeamCollection* self) {
 	}
 }
 
+ACTOR Future<Void> zeroServerLeftLogger_impl(DDTeamCollection* self, Reference<TCTeamInfo> team) {
+	wait(delay(SERVER_KNOBS->DD_TEAM_ZERO_SERVER_LEFT_LOG_DELAY));
+	state vector<KeyRange> shards = self->shardsAffectedByTeamFailure->getShardsFor(
+	    ShardsAffectedByTeamFailure::Team(team->getServerIDs(), self->primary));
+	state std::vector<Future<StorageMetrics>> sizes;
+	sizes.reserve(shards.size());
+
+	for (auto const& shard : shards) {
+		sizes.emplace_back(brokenPromiseToNever(self->getShardMetrics.getReply(GetMetricsRequest(shard))));
+		TraceEvent(SevError, "DDShardLost").detail("ShardBegin", shard.begin).detail("ShardEnd", shard.end);
+	}
+
+	wait(waitForAll(sizes));
+
+	int64_t bytesLost = 0;
+	for (auto const& size : sizes) {
+		bytesLost += size.get().bytes;
+	}
+
+	TraceEvent(SevError, "DDZeroServerLeftInTeam").detail("Team", team->getDesc()).detail("TotalBytesLost", bytesLost);
```

Comment:
We should give some thought to using `SevError` here, as there are other consequences to doing so. In particular, this process (and the cluster) will be marked unhealthy until restarted, which would be undesirable for a temporary issue. This could arguably be a justification for modifying the semantics of `SevError` and how we report it through status, but historically we've treated `SevError` as something leading to the death of the process (or possibly the role containing it). `SevWarnAlways` would be the safer play if we didn't want interaction effects with other features.

Reply:
Thanks for pointing this out. Will change to 'SevWarnAlways' instead.